R/helpers.R

```diff
@@ -39,3 +39,8 @@ check_subsetpattern = function(x, choices, empty.ok = TRUE) { # nolint
       paste0("{", paste0(x, collapse = ", "), "}")))
   }
 }
+
+get_akritas_learner = function() {
+  require_namespaces("mlr3extralearners")
```

Comment:
It's crazy this doesn't trigger an rmcdcheck NOTE to add this to Suggests...seems like a loophole that shouldn't be there! I'm not complaining...